doc/programming_guide/asrc/software_architecture.rst

```diff
@@ -0,0 +1,224 @@
+
+*********************
+Software Architecture
+*********************
+
+The ASRC demo application is a two tile application developed to run on the XK-VOICE-L71 board running at a core frequency of 600 MHz.
+
+It is a FreeRTOS based application where all the application blocks are implemented as FreeRTOS tasks.
+
+Each tile has 5 bare metal cores dedicated to running RTOS tasks and since all processing is done within RTOS tasks, each core has 120 MHz of bandwidth
+available.
+
+
+Task diagram
+============
+
+The :ref:`fig-ASRC-task-diagram-label` shows the RTOS tasks and other components that make up the system.
+
+.. _fig-ASRC-task-diagram-label:
+
+.. figure:: diagrams/asrc_task_diagram.drawio.png
+   :align: center
+   :scale: 80 %
```

Comment:
Needs to change to `:width: 100%` here and elsewhere.

Reply:
done